tooling/templatize/pkg/pipeline/run.go

```diff
@@ -100,7 +77,7 @@ func RunPipeline(pipeline *Pipeline, ctx context.Context, options *PipelineRunOp
 	if err != nil {
 		return nil, err
 	}
-	dir := filepath.Dir(pipeline.pipelineFilePath)
+	dir := filepath.Dir(options.PipelineFilePath)
 	logger.V(7).Info("switch current dir to pipeline file directory", "path", dir)
 	err = os.Chdir(dir)
```

Comment:
n.b. `os.Exec` has a place to set the working directory of the commands, wonder why you needed to mutate program state like this.

Reply:
I'll note this for additional refactoring